easy-ui-react/src/Menu/Menu.module.scss

```diff
@@ -0,0 +1,99 @@
+@use "../styles/common" as *;
+
+.underlay {
+  position: fixed;
+  inset: 0;
+}
+
+.root {
+  @include component-token(
+    "menu",
+    "border_radius",
+    design-token("shape.border_radius.base")
+  );
+  @include component-token(
+    "menu",
+    "color.background",
+    theme-token("color.surface.overlay")
+  );
+  @include component-token("menu", "color.text", theme-token("color.text"));
+  @include component-token(
+    "menu",
+    "color.text.disabled",
+    theme-token("color.text.disabled")
+  );
+  @include component-token(
+    "menu",
+    "color.background.hovered",
+    theme-token("color.surface.overlay.hovered")
+  );
+  @include component-token("menu", "shadow", theme-token("shadow.overlay"));
+  @include component-token("menu", "padding.x", design-token("space.2"));
+  @include component-token("menu", "color.border", theme-token("color.border"));
+
+  @include responsive-prop("menu", "width", width);
+
+  background: component-token("menu", "color.background");
+  border: 1px solid component-token("menu", "color.border");
+  border-radius: component-token("menu", "border_radius");
```

Comment:
**question**: out of curiosity is there a benefit to defining the border as `1px solid component-token("menu", "color.border");` and on the next line explicitly defining the `border-radius`?

Reply:
hm i'm not sure i understand the question. what's the alternative that you're thinking of?

Reply:
derp, i just realized i confused `border-radius` and `border-width` lol, so ignore that question, but could we use the `border_width` token here?

Reply:
oh good call. we can